src/getHumanAge.test.js

```diff
@@ -3,8 +3,39 @@
 describe('getHumanAge', () => {
   const { getHumanAge } = require('./getHumanAge');
 
-  test('should be declared', () => {
-    expect(getHumanAge)
-      .toBeInstanceOf(Function);
+  test('should return [0, 0] for catAge = 0 and dogAge = 0', () => {
+    expect(getHumanAge(0, 0)).toEqual([0, 0]);
+  });
+
+  test('should return [1, 1] for catAge = 15 and dogAge = 15', () => {
+    expect(getHumanAge(15, 15)).toEqual([1, 1]);
+  });
+
```

Comment:
Please also check 2 years and boundaries for this value

Reply:
Why did you remove this test?) you need to add more, should be at least all boundaries, for example: getHumanAge(14, 14), getHumanAge(15, 15), getHumanAge(27, 27), getHumanAge(28,28)